### PR TITLE
[RUM-998] Introduce Github issue forms

### PR DESCRIPTION
**Pull request by plousada**
### What does this PR do?

Replaces the current markdown files with more detailed yaml based forms.

Created forms for:
- Bug Report
- Crash Report
- Feature Request
- Setup Issue
- Question

Using forms will create a more pleasant and organized UI versus just using markdown files.
Screenshot 2024-02-20 at 11 25 23

### Motivation

The main goal is to have more detailed issues and cut down on time spent gathering information.

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BugReport.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        For a more complex setup consider creating a small app that showcases the problem. 
+        **Note** - Avoid sharing any business logic, credentials or tokens.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: SDK logs
+      description: |
+        Please provide console logs before, during and after the bug occurs.
```
Comment (xgouchet) [inline .github/ISSUE_TEMPLATE/BugReport.yml:32]:
```suggestion
        Please provide telnet debug console logs before, during and after the bug occurs. 
```

File: .github/ISSUE_TEMPLATE/CrashReport.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Report crashes caused by the SDK. Please try to be as detailed as possible. 
+        Before getting started, if the problem is urgent please use our [official support channel](https://www.datadoghq.com/support/).
+  - type: textarea
+    id: stacktrace
+    attributes:
+      label: Stack trace 
+      description: Please provide us with the stack trace of the crash or a crash report.
```
Comment (xgouchet) [inline .github/ISSUE_TEMPLATE/CrashReport.yml:14]:
```suggestion
      description: Please provide us with the backtrace of the crash or a crash report.
```

File: .github/ISSUE_TEMPLATE/BugReport.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      label: Integration Method
+      options:
+        - ROPM
+        - Manual
+  - type: input
+    id: roku_version
+    attributes:
+      label: RokuOS Version
+      description: e.g. RokuOS 12.5            
+  - type: textarea
```
Comment (xgouchet) [inline .github/ISSUE_TEMPLATE/BugReport.yml:77]: Add the brighterscript version if used (marked as optional)

```yml
  - type: input
    id: brighterscript_version
    attributes:
      label: BrighterScript compiler version (optional)
      description: e.g. BrighterScript 0.65.22
    validations:
      required: false
```

File: .github/ISSUE_TEMPLATE/CrashReport.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  - type: dropdown
+    id: checked_lastest_sdk
+    attributes:
+      label: Does the crash manifest in the latest SDK version?
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true 
+  - type: textarea
```
Comment (xgouchet) [inline .github/ISSUE_TEMPLATE/CrashReport.yml:57]: Missing Roku OS and BrighterScript versions

```yml
  - type: input
    id: roku_version
    attributes:
      label: RokuOS Version
      description: e.g. RokuOS 12.5  
  - type: input
    id: brighterscript_version
    attributes:
      label: BrighterScript compiler version (optional)
      description: e.g. BrighterScript 0.65.22
    validations:
      required: false
```

File: .github/ISSUE_TEMPLATE/SetupIssue.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      label: Integration Method
+      options:
+        - ROPM
+        - Manual
+  - type: input
+    id: roku_version
+    attributes:
+      label: RokuOS Version
+      description: e.g. RokuOS 12.5            
+  - type: textarea
```
Comment (xgouchet) [inline .github/ISSUE_TEMPLATE/SetupIssue.yml:54]: Missing  BrighterScript info
```yml
  - type: input
    id: brighterscript_version
    attributes:
      label: BrighterScript compiler version (optional)
      description: e.g. BrighterScript 0.65.22
    validations:
      required: false
```

**Review by xgouchet — COMMENTED** [review]
Nice, just a few Roku specifics changes :)

File: .github/ISSUE_TEMPLATE/CrashReport.yml (hunk 4 above)
Comment (plousada) [reply .github/ISSUE_TEMPLATE/CrashReport.yml:57]: I was wondering if RokuOS made sense in this case, as a crash could span multiple RokuOS versions.

**Review by xgouchet — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/CrashReport.yml (hunk 4 above)
Comment (xgouchet) [reply .github/ISSUE_TEMPLATE/CrashReport.yml:64]: It still makes sense IMHO as if it spans multiple versions, they can add them all there.

**Review by xgouchet — APPROVED** [review]

**Pull request merged**